scripts/generate-locale-messages.js

```diff
@@ -43,7 +43,7 @@ const fs = require('fs');
 const path = require('path');
 const mkdirp = require('mkdirp');
 
-const locales = ['en', 'es', 'fr'];
+const locales = ['en', 'es', 'de', 'ar', 'he'];
```

Comment:
For the purposes of review, ignore the changes having to do with the list of locales, it's changed so we can test RTL and generally longer translations. And this will all go away once it gui imports scratch-l10n.